### DISCUSSION TITLE
forge create -: error trying to connect: tcp connect error: Connection refused (os error 111) ??

### DISCUSSION THREAD

**Discussion by 0xconstructor** [Cyfrin/foundry-full-course-cu#28] [Q&A] [answered]
help me, someone, to resolve this issue I can't get the desired output even though everything I have tried 
something is wrong with the TCP connection error establishment how do I resolve and deploy at that rpc URL 
![Screenshot (38)](https://github.com/Cyfrin/foundry-full-course-f23/assets/123366816/2b9e4f14-bbb4-40c0-8410-19345fb613a9)

**Comment by cromewar** [comment, marked as answer]
Hello @web3baji 
please go to ganache config, under the  host config an change it to WSL, then use the new IP address.

**Reply by 0xconstructor** [reply]
![Screenshot (40)](https://github.com/Cyfrin/foundry-full-course-f23/assets/123366816/7d28fe6d-ff9f-40f0-9edc-cf4029d7a537)

are you referring to this??

**Reply by cromewar** [reply]
Yes sir, under hostname you will find one called WSL.

**Reply by 0xconstructor** [reply]
Yeah, the problem is solved.
thank you for helping me.

**Reply by satyam455** [reply]
hey im using anvil how to solve this issue??

**Comment by alymurtazamemon** [comment]
@web3baji Which one are you using Ganache or Anvil? If Ganache then make sure it is running.

**Comment by PatrickAlphaC** [comment]
Can you go to your bash terminal where anvil is running? I'll willing to bet your port is off. 


<img width="225" alt="Screenshot 2023-06-09 at 11 40 49 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/54278053/0f3f6024-7173-4fbe-9b54-43d67b57b159">

**Reply by 0xconstructor** [reply]
yepp!! deploying through anvil works fine , but when it comes to ganache it shows that output as connection refused

**Reply by PatrickAlphaC** [reply]
Great! yeah just use anvil for now :)
